apps/web/src/state/info/hooks.ts

```diff
@@ -136,14 +136,14 @@ export const usePoolTransactionsSWR = (address: string): Transaction[] | undefin
 
 // Tokens hooks
 
-export const useAllTokenHighLight = (targetChainName?: MultiChainNameExtend): TokenData[] => {
+export const useAllTokenHighLight = (enable: boolean, targetChainName?: MultiChainNameExtend): TokenData[] => {
```

Comment:
Can we turn the arguments into an object?
```suggestion
export const useAllTokenHighLight = (options: { enable?: boolean; targetChainName?: MultiChainNameExtend }): TokenData[] => {
```